design-doc-template.adoc

```diff
@@ -44,36 +44,48 @@ promoted-by:
 :idprefix:
 :idseparator:       -
 
-__<The entire document should be one to two pages long. We will write each analysis document as if it is a conversation
+////
+The entire document should be one to two pages long. We will write each analysis document as if it is a conversation
 with a future developer. This requires a good writing style, with full sentences organized into paragraphs. Bullets are
-acceptable only for visual style, not as an excuse for writing sentence fragments.>__
+acceptable only for visual style, not as an excuse for writing sentence fragments.
+////
```

Comment:
Keep as a visible comment and force the writer to delete so reader doesn't have to.